### PR TITLE
Adding repository naming conventions

### PR DESCRIPTION
**Pull request by johntkeller**
Take a look @Izzmo !

### CODE REVIEW

**Comment by Izzmo** [comment]
Added the rest of the team as well.

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (Izzmo) [inline .vs/VSWorkspaceState.json]: @johntkeller we do not want to include the ".vs" folder. I think we should add that to the .gitignore.

File: README.md [hunk 2]
```diff
@@ -11,11 +11,12 @@ This information can be updated quite often, but is all peer reviewed. If you se
 * [Feature Toggles](process/feature-toggles.md)
 
 
-## Coding Standards
+## [Coding Standards](/codingstandards)
```
Comment (Izzmo) [inline README.md:14]: Do you think we should link headings, or instead, link each item under the heading?

File: codingstandards/repositorynamingconvention.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+injection with more test-friendly implementations.  The interface should be named with the "I" prefix according
+to the coding standard, followed by the primary entity that it manages for its CRUD operations.
+
+## Method Names
+Methods for CRUD operations follow very consistent patterns, so it is extremely helpful to the callers of
+our repository classes to have consistent names, indicating what the behavior is exactly.
+
+Below is an example to reference as we explain the naming convention.
+
+    // Interface for the persistence layer of orders.
```
Comment (Izzmo) [inline codingstandards/repositorynamingconvention.md:16]: You can give type hinting in here by doing a triple tick and the type, so in this case, "```csharp" followed by your code, and ending in a triple tick.

```csharp
// Interface test..
public interface IOrder
{
}
```

**Review by Izzmo — CHANGES_REQUESTED** [review]

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (johntkeller) [reply .vs/VSWorkspaceState.json]: @Izzmo I figured that was all set up.  I'll make that change.

**Review by Izzmo — APPROVED** [review]

**Review by guzumba15 — APPROVED** [review]

**Review by NavathaP — APPROVED** [review]

**Pull request merged**